docs/source/python_api/mlflow.metrics.rst

```diff
@@ -160,6 +160,7 @@ Parameters:
 
 .. autofunction:: mlflow.metrics.recall_at_k
 
+.. autofunction:: mlflow.metrics.ndcg_at_k
```

Comment:
I think you'll have to add to the `Retriever Metrics` blurb above for ndcg as well!
ex. [linked here](https://github.com/mlflow/mlflow/pull/10144/files#diff-ef4ee65c3d24eaada583650c0f1004db2c67b688c0249cf754df6e65dfd0aa7eR145-R155)

Reply:
Updated above